.github/workflows/auto-merge-dependabot-pr.yaml

```diff
@@ -4,7 +4,10 @@
 name: Auto-merge Dependabot PR
 
 # Run when a pull request is opened or updated
-on: pull_request
+on:
+  pull_request:
+    branches:
+      - "dependabot/**"
```

Comment:
Dependabot can create branches with multiple `/` in them, hence the double star.